packages/website/blog/2024-05-27-announcing-typescript-eslint-v8-beta.mdx

```diff
@@ -273,14 +276,15 @@ See [feat(utils): allow specifying additional rule meta.docs in RuleCreator](htt
 - ⏳ [Repo: Rule [options] parameter should be non-nullable if defaultOptions exists](https://github.com/typescript-eslint/typescript-eslint/issues/5439)
 - [feat(parser): always enable comment, loc, range, tokens](https://github.com/typescript-eslint/typescript-eslint/pull/8617)
   - If you were manually calling `@typescript-eslint/parser` functions, those options are no longer necessary to provide
-- ⏳ [feat(rule-tester): support multipass fixes](https://github.com/typescript-eslint/typescript-eslint/pull/8883)
+- [feat(rule-tester): support multipass fixes](https://github.com/typescript-eslint/typescript-eslint/pull/8883)
```

Comment:
Does this have a note in this blog post at all? It might be complex enough to require its own section just showing a before/after example (can grab one from the PR for a concrete example)